README.md

```diff
@@ -29,9 +29,35 @@ I did this ages ago, so I don't really remember.
 
 1.  Install `gnome-common`
 2.  Install the `python3-arrow`  and `python3-markdown`
-3.  Install a SCSS compiler (eg. `rubygem-sass` on Fedora).  This is very
+3.  Install `libglib2.0-dev`,`libgtk-3-dev`,`libsoup2.4-dev`,`libjson-glib-dev`
+4.  Install a SCSS compiler (eg. `rubygem-sass` on Fedora).  This is very
```

Comment:
Other SCSS compilers do work. Please do not recommend a tool that needs installing additional dependencies.